sdk/src/types/utils.ts

```diff
@@ -52,3 +52,7 @@ export type NestedPartial<Item extends object> = {
  * Resolve type to its final object
  */
 export type Identity<U> = U extends infer A ? A : U;
+
+export type Prettify<T> = {
```

Comment:
Let's add a JSDoc description:

```suggestion
/**
 * Reduces a complex object type to make it readable in IDEs.
 */
export type Prettify<T> = {
```